config/initializers/foreman_register.rb

```diff
@@ -40,24 +40,24 @@
     common_th_class = 'hidden-tablet hidden-xs'
     common_td_class = common_th_class + ' ellipsis'
     add_pagelet :hosts_table_column_header, key: :model, label: _('Model'), sortable: true, width: '10%', class: common_th_class
-    add_pagelet :hosts_table_column_content, key: :model, class: common_td_class, callback: ->(host) { host.compute_resource_or_model }
+    add_pagelet :hosts_table_column_content, key: :model, class: common_td_class, callback: ->(host) { host.compute_resource_or_model }, export_key: 'compute_resource_or_model'
```

Comment:
I think it would be more correct to have the `export_key` at the column header level, rather than data level. This way you can also utilize the `:label` to generate the column names in the header of the CSV.

Reply:
Labels are translated, wouldn't it be odd to have localized csvs? For example my bank does that and its terrible